src/test/java/com/puppycrawl/tools/checkstyle/CheckerTest.java

```diff
@@ -93,18 +100,29 @@ private static Method getFireAuditStartedMethod() throws NoSuchMethodException {
         return fireAuditStarted;
     }
 
+    private static Method getAcceptFileStartedMethod() throws NoSuchMethodException {
+        final Class<Checker> checkerClass = Checker.class;
+        final Method acceptFileStarted =
+            checkerClass.getDeclaredMethod("acceptFileStarted", String.class);
+        acceptFileStarted.setAccessible(true);
+        return acceptFileStarted;
+    }
```

Comment:
get rid of this method, 
If you can do testing withtout reflection - so you have NOT use it.
Reflection is ok only when no other ways to do this.
But all code any class should be accessible by calls of public methods.